### ISSUE TITLE
spfx 1.13 and PropertyFieldListPicker context 

### ISSUE THREAD

**Issue by Link631** [pnp/sp-dev-fx-property-controls#444]
#### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question

#### Version
"@pnp/spfx-property-controls": "^3.6.0
spfx 1.13.1

#### Expected / Desired Behavior / Question
when using latest spfx with latest spfx property controls, it should be possible to pass the webpartContext property.

#### Observed Behavior
when using latest spfx with latest spfx property controls, following error appears, when I want to use the PropertyFieldListPicker:
Type 'WebPartContext' is missing the following properties from type 'BaseComponentContext': _isServedFromLocalhost, isServedFromLocalhost

Do you have the same behaviour?

Thanks....

**Comment by jimmywim** [comment]
Yeah we've noticed the same thing. Does 3.6 require a newer version of spfx? Should this be detailed in the docs?

**Comment by milanholemans** [comment]
Was discussed at [#434](https://github.com/pnp/sp-dev-fx-property-controls/issues/434).
